src/fixtures/local/complex/.keboola/manifest.json

```diff
@@ -19,21 +19,21 @@
       "branchId": %%LOCAL_STATE_MAIN_BRANCH_ID%%,
       "componentId": "ex-generic-v2",
       "id": "%%LOCAL_STATE_GENERIC_CONFIG_ID%%",
-      "path": "ex-generic-v2/456-todos",
+      "path": "extractor/ex-generic-v2/456-todos",
```

Comment:
V niektorych testoch chybal v ceste ku suborom typ komponenty, napr. `extractor`. 
Tieto testy vznikli skor, ked to bolo implementovane bez typu komponenty v `path`.

V manifeste moze byt lubovolna cesta ku existujucemu suboru, ... no zjednotil som to, aby som `src/fixtures/local/...` mohol pouzivat aj v novych testoch a bolo to jednotne.